tests/test_misc.lua

```diff
@@ -541,4 +542,382 @@ T['zoom()']['respects `config` argument'] = function()
   child.expect_screenshot()
 end
 
+T['setup_restore_cursor()'] = new_set()
+
+local setup_restore_cursor = function(...) child.lua('MiniMisc.setup_restore_cursor(...)', { ... }) end
+
+local restore_cursor_default_opts = {
+  ignore_buftype = { 'nofile' },
+  ignore_filetype = { 'gitcommit' },
+  center = true,
+}
+
+T['setup_restore_cursor()']['autocmd'] = function()
+  setup_restore_cursor()
+
+  eq(child.lua_get([[#vim.api.nvim_get_autocmds({ group = 'MiniMiscRestoreCursor' })]]) > 0, true)
```

Comment:
This will break on Neovim<0.7. Use `eq(child.fn.exists('#MiniMiscRestoreCursor'), 1)`

Reply:
Will do, but this is copy & paste from the same file.

Reply:
> Will do, but this is copy & paste from the same file.

Yes, because there is doesn't run on Neovim<0.7 . But still, should probably change it there.